.gitignore

```diff
@@ -36,7 +36,7 @@ node_modules
 /.sass-cache
 /.scannerwork
 /connect.lock
-/coverage
+coverage
```

Comment:
Nit: Why? If this needs changing, do the other entries here need changing too?

Reply:
I'm assuming you added coverage since lately I started getting these every time I executed tests.
![image](https://github.com/ag-grid/ag-charts/assets/1083065/019e0b40-a4b2-439a-abac-f95499157415)
I looked at the other entries and it didn't seem relevant to change any of them.